Detections/MultipleDataSources/IridiumIOCs.yaml

```diff
@@ -0,0 +1,121 @@
+id: 7ee72a9e-2e54-459c-bc8a-8c08a6532a63
+name: Known IRIDIUM IP
+description: |
+  'IRIDIUM command and control IP. 
+   Matches  IP IOCs related to the IRIDIUM activity group with CommonSecurityLog, OfficeActivity, DnsEvents, VMConnection and SecurityEvents dataTypes.'
```

Comment:
Extra space after "Matches".  Also, we should stay with our standard "Identifies" and spell out IPs with IP Addresses along with IOCs with Indicators of Comprise (IOCs).  Lastly, no need to list the datatypes, that is clear below and also they are different.  Also, I guess we did not get a public blog link yet?

Reply:
Thanks!!
Removed spaces after "Matches"
Changed the description to align with the "Identifies" format.
Removed Data Types.
For this one it seems no Blog post is coming out.